### PR TITLE
Add a 'why' paragraph to the docs (Kinto value proposition)

### LINKED ISSUE #410: Kinto value proposition -- why do I need a JSON store?
Why do I need a dedicated JSON store when databases, such as postgresql, support JSON and JSONB?

### PR DESCRIPTION
**Pull request by magopian**
Fixes #410

Landing page:
screen shot 2016-03-01 at 14 53 52

Overview page, first paragraph:
screen shot 2016-03-01 at 14 54 20

### CODE REVIEW

File: docs/overview.rst [hunk 1]
```diff
@@ -10,6 +10,32 @@ abilities. It is meant to be **easy to use** and **easy to self-host**.
...
+
+We believe data belongs to the users, and not to the application authors. When
+writing an application, data should be made available to any device, connected
+or offline, and everything should be in sync.
+
+Rather than spending a non-trivial amount of time and expertise on implementing
+all that (and then maintaining it!), you could use Kinto, that does all that
+for you:
+
+- Expose your data over an HTTP interface, unlike a database like PostreSQL
```
Comment (leplatrem) [inline docs/overview.rst]: typo: PostgreSQL

File: docs/overview.rst [hunk 2]
```diff
@@ -10,6 +10,32 @@ abilities. It is meant to be **easy to use** and **easy to self-host**.
...
+- Use simple HTTP queries, instead of SQL
```
Comment (leplatrem) [inline docs/overview.rst]: queries ? requests ?

**Comment by Natim** [comment]
LGTM :+1:

File: docs/overview.rst [hunk 3]
```diff
@@ -10,6 +10,32 @@ abilities. It is meant to be **easy to use** and **easy to self-host**.
...
+- Use Kinto.js to easily implement offline first clients
+- Choose the database you want from those that Kinto supports, and use a
+  unified API to access its data
+- Manage your data using the handy admin UI
+- Easily set up live push notifications for live synchronisation
+- Make it possible to share data between users (using fine-grained permissions)
+- Take advantage of schema validation if you need it
+- Use tools and libraries like Kinto.js that implements encryption so you don't
```
Comment (leplatrem) [inline docs/overview.rst]: I'm bit embarassed by this since Kinto.js does not provide anything specific for encryption :)

Comment (magopian) [inline docs/overview.rst]: Should it be reworded to "Use tools and libraries like Kinto.js that help encrypt your data"?

Comment (magopian) [inline docs/overview.rst]: Or maybe "Use tools and libraries, like Kinto.js and its remote transformers that make encrypting your data a breeze"?

Comment (almet) [inline docs/overview.rst]: We probably should just remove this right now, and fix it in Kinto.js, so we can support encryption there :)

**Comment by magopian** [comment]
I added @leplatrem's wording/typo fixes, and removed the problematic sentence, what do you think @leplatrem @almet ?

File: docs/overview.rst [hunk 4]
```diff
@@ -10,6 +10,30 @@ abilities. It is meant to be **easy to use** and **easy to self-host**.
 *Kinto* is used at Mozilla and released under the Apache v2 licence.
 
 
+.. _why:
+
+Why
```
Comment (almet) [inline docs/overview.rst]: Why what ? Why are we building kinto ? Why should you use kinto ?

File: docs/overview.rst [hunk 5]
```diff
@@ -10,6 +10,30 @@ abilities. It is meant to be **easy to use** and **easy to self-host**.
...
+===
+
+We believe data belongs to the users, and not to the application authors. When
+writing an application, data should be made available to any device, connected
```
Comment (almet) [inline docs/overview.rst]: s/an application/applications

File: docs/overview.rst [hunk 6]
```diff
@@ -10,6 +10,30 @@ abilities. It is meant to be **easy to use** and **easy to self-host**.
...
+or offline, and everything should be in sync.
+
+Rather than spending a non-trivial amount of time and expertise on implementing
+all that (and then maintaining it!), you could use Kinto, that does all that
+for you:
+
+- Expose your data over an HTTP interface, unlike a database like PostgreSQL
```
Comment (almet) [inline docs/overview.rst]: unlike database**s**

File: docs/overview.rst [hunk 7]
```diff
@@ -10,6 +10,30 @@ abilities. It is meant to be **easy to use** and **easy to self-host**.
...
+- Use simple HTTP requests, instead of SQL
```
Comment (almet) [inline docs/overview.rst]: the "," is superfluous here. Should we replace "instead" by "rather"?

File: docs/overview.rst [hunk 8]
```diff
@@ -10,6 +10,30 @@ abilities. It is meant to be **easy to use** and **easy to self-host**.
...
+- Use Kinto.js to easily implement offline first clients
```
Comment (almet) [inline docs/overview.rst]: Add a link to Kinto.js

File: docs/overview.rst [hunk 9]
```diff
@@ -10,6 +10,30 @@ abilities. It is meant to be **easy to use** and **easy to self-host**.
...
+- Choose the database you want from those that Kinto supports, and use a
+  unified API to access its data
+- Manage your data using the handy admin UI
```
Comment (almet) [inline docs/overview.rst]: link to the admin UI

File: docs/overview.rst [hunk 10]
```diff
@@ -10,6 +10,30 @@ abilities. It is meant to be **easy to use** and **easy to self-host**.
...
+- Easily set up live push notifications for live synchronisation
```
Comment (almet) [inline docs/overview.rst]: link to the section about this in the docs

File: docs/overview.rst [hunk 11]
```diff
@@ -10,6 +10,30 @@ abilities. It is meant to be **easy to use** and **easy to self-host**.
...
+- Make it possible to share data between users (using fine-grained permissions)
```
Comment (almet) [inline docs/overview.rst]: link to the docs

File: docs/overview.rst [hunk 12]
```diff
@@ -10,6 +10,30 @@ abilities. It is meant to be **easy to use** and **easy to self-host**.
...
+- Take advantage of schema validation if you need it
```
Comment (almet) [inline docs/overview.rst]: ditto

**Comment by almet** [comment]
That's great I love it :) We should add a few more links and be good to go!

**Comment by almet** [comment]
That's great!

**Comment by almet** [comment]
thanks :)

**Pull request merged**